src/components/ui/HeaderLogo.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import logo from '../../assets/logo.png';
+
+const HeaderLogo:React.FC = () => {
+  return (
+    <div >
```

Comment:
Can directly send img tag without the div if the div is not used to trigger a style in css

Reply:
removed